prac_06/programming_language.py

```diff
@@ -0,0 +1,18 @@
+"""
+Programming language
+Estimate: 30 minutes
+Current time: 10:25
+Actual:   50 minutes
+"""
+class ProgrammingLanguage:
```

Comment:
The class is concise, with attributes that match the context of a programming language (name, typing, reflection, year).